slob.cc

```diff
@@ -414,7 +415,7 @@ const SlobFile::RefOffsetsVector & SlobFile::getSortedRefOffsets()
       refsOffsetVector.append( RefEntryOffsetItem( base + qFromBigEndian( tmp ), i ) );
     }
 
-    qSort( refsOffsetVector );
+    std::sort( refsOffsetVector );
```

Comment:
Fixing deprecation warnings is nice. But did you try compiling this? `std::sort` requires at least two arguments. The same issue in other places.

Reply:
> But did you try compiling this?

OMG, I didn't notice any compilation errors and ran the old binary. :(
I'll fix it soon™.